README.md

```diff
@@ -58,7 +58,10 @@ If you have chosen the first option, you can follow the theme reference instruct
 Following the point #2 you will need to upload your theme using theme installer.
 
 - Ensure that your root directory does not contain any .git hidden folders or files
+- Follow the base theme folder directory even when the folder are empty
```

Comment:
I am not sure how true this is. I was able to upload a theme missing the /asset/src folder. Do we really need all folders despite being empty?